### PR TITLE
Add some annotations

### PR DESCRIPTION
**Pull request by ankur2136**
#164

### CODE REVIEW

File: android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/DbContentProvider.java [hunk 1]
```diff
@@ -19,42 +20,50 @@
         this.mDb = db;
     }
 
-    protected abstract T cursorToEntity(Cursor cursor);
+    protected abstract T cursorToEntity(@NonNull Cursor cursor);
 
-    protected long insert(String tableName, ContentValues values) {
+    @IntRange(from = -1)
+    protected long insert(@Nullable String tableName, @NonNull ContentValues values) {
```
Comment (sdlaver) [inline android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/DbContentProvider.java:26]: `tableName` should probably be `@NonNull`?

File: android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/DbContentProvider.java [hunk 2]
```diff
@@ -19,42 +20,50 @@
...
                 selectionArgs, null, null, sortOrder, limit);
     }
 
-    protected Cursor query(String tableName, String[] columns,
-                        String selection, String[] selectionArgs, String groupBy,
-                        String having, String orderBy, String limit) {
+    @NonNull
+    protected Cursor query(@NonNull String tableName, @Nullable String[] columns,
+                           @Nullable String selection, @Nullable String[] selectionArgs,
+                           @Nullable String groupBy,
```
Comment (sdlaver) [inline android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/DbContentProvider.java:56]: NIT: whitespace across lines 56 and 57 seems off

File: android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/PublicKeysDao.java [hunk 3]
```diff
@@ -21,7 +21,7 @@ public class PublicKeysDao extends DbContentProvider<PublicKey> implements Publi
     }
 
     @Override
-    protected PublicKey cursorToEntity(Cursor cursor) {
+    protected PublicKey cursorToEntity(@NonNull Cursor cursor) {
```
Comment (sdlaver) [inline android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/PublicKeysDao.java:24]: It looks like the return value from this function is always non-null; can we annotate it as such?

**Review by sdlaver — CHANGES_REQUESTED** [review]

File: android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/DbContentProvider.java (hunk 1 above)
Comment (ankur2136) [reply android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/DbContentProvider.java]: Oops. Fixed.

**Review by sdlaver — APPROVED** [review]

**Pull request merged**